utils/luci-app-owm/files/owm.lua

```diff
@@ -53,6 +53,15 @@ function unlock()
 	os.execute("rm -f "..lockfile)
 end
 
+-- a location for the node is required to run
+if next(owm.get_position())==nil then
```

Comment:
Maybe `owm.get_position()` should just return `nil` if we can't get the position from the configuration file? This `next` call is a bit irritating.